packages/orbit-tailwind-preset/README.md

```diff
@@ -23,12 +23,17 @@ The `orbitComponentsPreset` function accepts an optional object with one propert
 In your `tailwind.config.js` file (or equivalent), add the following:
 
 ```js
+const path = require("node:path");
 const orbitComponentsPreset = require("@kiwicom/orbit-tailwind-preset");
+// the following ensures the correct path is found (especially within monorepos)
+const orbitComponentsPath = require
```

Comment:
I know that it was documented as CSJ before, but wdyt, maybe the better way will be to document that right with *ts* with the ESM example? Our users use typescript mainly and the config will be most likely in ts 🤔

Reply:
I'd say it's out of scope for this PR.

The only point was to make the instructions work with monorepos.

If we want to use a different format and extension, let's do it separately.